packages/launcher/src/renderer/launcher/developer/PermissionsRequirements.js

```diff
@@ -192,31 +228,29 @@ export default class PermissionsRequirementsView extends Component<
         )
       }
 
-      const onToggle = (newState: PermissionRequirement) => {
-        this.setState(({ permissionSettings }) => {
-          if (newState === permissionSettings[key]) {
-            delete permissionSettings[key]
-          } else {
-            permissionSettings[key] = newState
-          }
-          return { permissionSettings }
-        })
-      }
-
       return (
         <View style={styles.permissionRow} key={key}>
           <Text style={styles.permissonDescription}>
             {PERMISSIONS_DESCRIPTIONS[key]}
           </Text>
-          {this.renderToggle(key, permissionSettings[key], onToggle)}
+          {this.renderToggle(key, permissionSettings[key], this.onToggle)}
         </View>
       )
     })
 
+    const errorLabel = this.state.errorMsg ? (
+      <Text style={[styles.errorLabel, globalStyles.errorText]}>
+        {this.state.errorMsg}
+      </Text>
+    ) : null
+
     return (
-      <View style={styles.container}>
+      <View>
```

Comment:
Not an issue but if you only need a `View` or other component as a way of containing multiple children, the new `Fragment` element in React is made for this, that allows to flatten the tree a bit.